tests/ctst/steps/utils/kubernetes.ts

```diff
@@ -165,6 +166,20 @@ export async function createJobAndWaitForCompletion(
     }
 }
 
+export async function createAndRunS3UtilsPod(
```

Comment:
This function can be more generic, it runs a pod given a manifest, not just s3utils